src/lib/mina_lib/mina_lib.ml

```diff
@@ -453,6 +453,21 @@ let get_node_state t =
   ; uptime_of_node
   }
 
+(** Compute the hard fork genesis slot from the runtime config, if all the stop
+    slots and the genesis slot delta have been set *)
+let scheduled_hard_fork_genesis_slot t :
```

Comment:
All though it's clear by the implementation, let's document a bit more that, the `hard_fork` in `scheduled_hard_fork_genesis_slot` is the next upcoming hard fork, and the `hard_fork` in the returning type `Mina_numbers.Global_slot_since_hard_fork.t` is the last hard fork.